src/components/components/faq-group/faq-group.js

```diff
@@ -13,9 +13,16 @@ export default props => {
     <div className={classes}>
       <ShadowBox>
         <div className="faq__group-header">
-          <h2 title="title title--x-small">{props.header}</h2>
+          <h2 title="title title--x-small" id={props.header.replace(/\s/g, "")}>
+            {props.header}
+          </h2>
         </div>
-        <ul className="faq__group-content">{props.children}</ul>
+        <ul
+          className="faq__group-content"
+          aria-labelledby={props.header.replace(/\s/g, "")}
```

Comment:
Could you safeguard the replace call here from null header?

Reply:
done